packages/click-to-react-component/src/ClickToComponent.js

```diff
@@ -55,15 +52,26 @@ export function ClickToComponent({ editor = 'vscode' }) {
   const onClose = React.useCallback(
     function handleClose(returnValue) {
       if (returnValue) {
-        const url = `${editor}://file/${returnValue}`
-        window.open(url)
+        openFileInIDE(returnValue)
       }
-
       setState(State.IDLE)
     },
     [editor]
   )
 
+  const openFileInIDE = (file) => {
+    if (editor === 'intellij') {
+      const url = `http://localhost:63342/api/file/${file}`
```

Comment:
TIL!  I'd like to move this logic up so that `lineNumber` & `columnNumber` can be used.

https://github.com/JetBrains/intellij-community/blob/a77365debaadcf00b888a977d89512f3f0f3cf9e/platform/built-in-server/src/org/jetbrains/ide/OpenFileHttpService.kt#L52-L59

This seems related to #28, #29, or #26, too 🤔

Reply:
I confirm that in the state it works well, intellij opens the file in the right row and in the right column.
Would you prefer to use another syntax?

Reply:
@ericclemmons JetBrain's IDEs support both formats: file.kt:100:34 and file.kt&line=100&column=34. No reason for additional logic here. It is just work.